From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Leandro Dorileo <leandro.maciel.dorileo@intel.com>
Date: Tue, 19 Nov 2019 13:39:44 -0800
Subject: [PATCH] fix debuginfo build-id matching code

We're relocating debuginfo content to /usr/share/debug with that
the rpm build code can't match .build-id files since it's trying
to match prefixed by /usr/lib/debug.

Signed-off-by: Leandro Dorileo <leandro.maciel.dorileo@intel.com>
---
 build/files.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/build/files.c b/build/files.c
index 2de6c9b..10b88ad 100644
--- a/build/files.c
+++ b/build/files.c
@@ -46,9 +46,9 @@
  */
 #define BUILD_ID_DIR		"/usr/lib/.build-id"
 #define DEBUG_SRC_DIR		"/usr/src/debug"
-#define DEBUG_LIB_DIR		"/usr/lib/debug"
-#define DEBUG_LIB_PREFIX	"/usr/lib/debug/"
-#define DEBUG_ID_DIR		"/usr/lib/debug/.build-id"
+#define DEBUG_LIB_DIR		"/usr/share/debug"
+#define DEBUG_LIB_PREFIX	"/usr/share/debug/"
+#define DEBUG_ID_DIR		"/usr/share/debug/.build-id"
 #define DEBUG_DWZ_DIR 		"/usr/lib/debug/.dwz"
 
 #undef HASHTYPE
